src/middleware.ts

```diff
@@ -1,5 +1,30 @@
 import { clerkMiddleware } from "@clerk/nextjs/server";
-export default clerkMiddleware();
+import { createClerkClient } from "@clerk/backend";
+import { NextRequest, NextResponse } from "next/server";
+
+const clerk = createClerkClient({
+    secretKey: process.env.CLERK_SECRET_KEY,
+});
+
+export default clerkMiddleware(async (auth, request: NextRequest) => {
+    const { userId, orgId, orgRole } = auth();
+    const pathname = request.nextUrl.pathname;
+    if (userId && pathname != "/manage-mfa") {
```

Comment:
How about making the URL just be `/mfa` and then have this check the start of the url  like `pathname.startsWith('/mfa')` vs strict equality. that way we can add other urls under it like `/mfa/add` or `/mfa/manage`

Reply:
i did originally have the path as `/accounts/mfa/add` or something. but through discussion with phil we simplified it, since this is just a demo